js/ng-views/edit.html

```diff
@@ -0,0 +1,7 @@
+<h3>Edit a 'gram</h3>
+<div>
+  <input type="text" value="vm.gram.photo_url" ng-model="vm.gram.photo_url">
```

Comment:
You don't need to use `value` and `ng-model`. `ng-model` alone will take care of populating the value when used on an `<input>`